packages/app/components/EventCard/EventCardText.js

```diff
@@ -48,29 +64,25 @@ function EventCardText(props) {
   return (
     <View>
       <Text style={styles.smallText}>
-        {props.timeBegin} - {props.timeEnd}
+        {formatDate(event.startDate)} - {formatDate(event.endDate)}
       </Text>
       <Text
         {...(isTitlePressed
           ? { style: styles.eventTitlePressed }
           : { style: styles.eventTitle })}
         onPress={onTitlePress}>
-        {props.title}
+        {event.title}
       </Text>
-      <Text style={styles.smallText}>📌 {props.location}</Text>
+      <Text style={styles.smallText}>📌 {event.location}</Text>
```

Comment:
A little thing you could do here, although i'd say it's optional, is if location and/or description is null,
hide their hide their text views or maybe set a default location/description such as N/A (former is probably better),
it makes the cards a little less empty.

Reply:
I also added this feature for start and end times.